dbt_semantic_interfaces/naming/dundered.py

```diff
@@ -91,52 +91,3 @@ def entity_prefix(self) -> Optional[str]:
             return DUNDER.join(tuple(entity_reference.element_name for entity_reference in self.entity_links))
 
         return None
-
-
-class DunderedNameFormatter:
```

Comment:
I'm assuming you checked that this isn't used in any dependent repos?

Reply:
Same for all the other deleted objects!

Reply:
yup!

Reply:
so `DunderedNameFormatter` is used in MF, but I thought it should be fine since this is release a minor version upgrade, which shouldn't be picked up by MF right away right?

Reply:
Sounds good! I would double check that `metricflow-semantics` won't pick it up (it's not always pinned to a patch version)

Reply:
Wait ignore me you're right. Minor is not the same is patch. Sorry my brain is busted rn 🙃